### ISSUE TITLE
Client side: Dynamic content on the home page

### ISSUE THREAD

**Issue by rasmus-storjohann-PG** [pg-irc/pathways-frontend#1151]
Replace the current, static COVID card with a new component that pulls dynamic content from the server so that we can push new content to the app without putting out a new version of the app. [This is the server side issue](https://github.com/pg-irc/pathways-backend/issues/527).

We'll call these new things "alerts".

Acceptance criteria:
- [x] Alerts appear on the app home screen, in the place where the COVID card is currently shown
- [x] The client requests alerts from the server. The locale is included in the request, and the response contains the content in the given locale.
- [x] The client can display zero or more alerts. For now, show all the alerts one above the next on the home screen. Probably we will only show a very small number of alerts, 0-2 maybe?
- [x] Each alert has a heading and a content string. 
- [x] The content string can contain mark-down, the same as the way our NCG content is shown in the topic detail component. 
- [x] The mark-down can contain links that are clickable to open in a browser.

**Comment by tomy-pg** [comment]
Staging version 1.34.0:

Header of dynamic content shows {0} instead of text.
![Screenshot_20200521-161621_Arrival Advisor](https://user-images.githubusercontent.com/47794846/82616367-d5e71880-9b81-11ea-957e-4bf0f135c8f7.jpg)

This is because of this line of code in `src/recommended_topics/call_to_action.tsx`:
```
<Text style={textStyles.headlineH2StyleBlackLeft}>
     <Trans>{alert.heading}</Trans>
</Text>
```
All that needs to happen is we should remove the wrapping `Trans` tag. We did not catch this in the smoke test because Lingui has a mechanism in development environments to use a fallback. In production it does not do this. This issue is the exact same issue I ran into here: https://github.com/pg-irc/pathways-frontend/issues/1057
